profiles/coreos/base/package.use

```diff
@@ -79,3 +79,5 @@ dev-cpp/glog gflags
 # enable rpc for rpc.rquotad
 sys-fs/quota rpc
 
+# don't ship the Samba daemons
+net-fs/samba minimal
```

Comment:
Since this is a cusomized ebuild if you use `+minimal` in IUSE it'll turn on by default.